Create Nest.js template

Graphback comes with number of templates (see repo templates directory.) Over the last month we seen increased demand to Nest.js framework and it wil.be good to be able to have some nest.js backend template

While setting up nest i am getting this error also tried to update node but didn't work

:\nest\node_modules\fs-extra\lib\mkdirs\make-dir.js:85
} catch {
^

SyntaxError: Unexpected token {
at createScript (vm.js:80:10)
at Object.runInThisContext (vm.js:139:10)
at Module._compile (module.js:607:28)
at Object.Module._extensions..js (module.js:654:10)
at Module.load (module.js:556:32)
at tryModuleLoad (module.js:499:12)
at Function.Module._load (module.js:491:3)
at Module.require (module.js:587:17)
at require (internal/module.js:11:18)
at Object. (D:\nest\node_modules\fs-extra\lib\mkdirs\index.js:3:44)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nest-typescript-starter@1.0.0 start: nest start
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nest-typescript-starter@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\hp\AppData\Roaming\npm-cache_logs\2020-12-06T12_19_21_261Z-debug.log`

--- a/packages/create-graphback/src/init/communityTemplates.ts
+++ b/packages/create-graphback/src/init/communityTemplates.ts
@@ -1,6 +1,6 @@
 
 /**
- * Array containing extrernal templates for the Graphback
+ * Array containing external templates for the Graphback
  * See contribution guide for more information
  */
 export const externalTemplates = [
@@ -11,6 +11,15 @@ export const externalTemplates = [
       uri: 'https://github.com/aerogear/graphback',
       branch: 'templates-1.0.0',
       path: '/templates/ts-apollo-mongo-apache-kafka-backend'
+    }]
+  },
+  {
+    name: 'nestjs-apollo-mongodb-backend',
+    description: 'Apollo GraphQL server using NestJS framework and connecting to MongoDB database',
+    repos: [{
+      uri: 'https://github.com/namit-chandwani/nestjs-apollo-mongodb-backend',
+      branch: 'master',
+      path: '/'
     }]
   }
 ];

